Update Quick Start Guide

CPKCampbell opened the pull request:
Update by convert-ly to 2.19.7
Shifted emphasis away from Windows to be more general
Updated links
Re-arranged some material
Many minor changes of punctuation and spelling

CPKCampbell committed:
Update Quick Start Guide

Update by convert-ly to 2.19.7
Shifted emphasis away from Windows to be more general
Updated links
Re-arranged some material
Many minor changes of punctuation and spelling

jan-warchol commented on uczenie/2 wprowadzenie do LilyPonda.ly:
```diff
@@ -26,29 +26,30 @@
-  \with-url #"http://github.com/wbsoft/frescobaldi/releases"
-  \smaller\typewriter"github.com/wbsoft/frescobaldi/releases",
+  \with-url #"http://code.google.com/p/lilykde/downloads/list"
+  \smaller\typewriter"code.google.com/p/lilykde/downloads/list",
```
As far as i can see the old link is the correct one - google code has only old versions.

CPKCampbell replied:
No idea how that brain cramp happened, Jan! I've rewritten the section to point to the Easier Editing section of the Introduction on lilypond.org.

jan-warchol commented on uczenie/2 wprowadzenie do LilyPonda.ly:
```diff
@@ -64,47 +65,54 @@
-  If after f you write c, you'll get a c that is closest to that f
+  The other way to enter pitches is by using relative mode. This uses the keyword  \typewriter "relative"
+  to specify a starting pitch. Following pitches are related to the note immediately preceding, and are placed as close as possible to it,
+  within a fourth below to a fifth above. For example, if after f you write c, you'll get the c which is closest to the f
```
it is within a fourth below and a fourth above.  Also, note that the comma after c is _very_ confusing.

CPKCampbell replied:
Re-written to expand the example a bit and eliminate the comma.

jan-warchol commented on uczenie/2 wprowadzenie do LilyPonda.ly:
```diff
@@ -64,47 +65,54 @@
 \markup \justify \italic {
-  Note that after letter f there's an apostrophe
-  (this is \bold not a sign from the key next to the digit 1).
+  Note that after the letter f there is an apostrophe:
```
Actually, isn't "single quote" the correct english name?

CPKCampbell replied:
They're usually equivalent, at least in ordinary English speech. They are actually different characters, but the single quote requires a multi-key press on most keyboards while the apostrophe is a single one. Regrettably, the documentation is inconsistent, using "apostrophe", "single quote", and "quote" interchangeably. I'll be arbitrary and use "single quote" as the least ambiguous name.

jan-warchol commented on uczenie/2 wprowadzenie do LilyPonda.ly:
```diff
@@ -112,64 +120,42 @@
-}
-
-{ \key f \major r2 b'2 }
+\markup \bold { Key signatures }
```
I think it would be best to introduce accidentals before key signatures.  Admittedly, previous version was not good because it started with a complicated example, but otoh this version is somewhat backwards.

CPKCampbell replied:
It seemed to me that notes are "accidental" only in relation to an established key signature. The need to specify the degree of a pitch only comes in when it alters the expected one, if you see what I mean. That's why I moved the key signature section before accidentals, moving from the general case to the exceptions.

jan-warchol commented on uczenie/2 wprowadzenie do LilyPonda.ly:
```diff
@@ -112,64 +120,42 @@
 
-\markup "key sig cheat-sheet:"
-\new Staff \with { \remove "Time_signature_engraver" } {
+% I'm not sure we really need the key sig cheat sheet
```
I remeber that someone asked me to include it.  But maybe there would be a better place for it.

CPKCampbell replied:
This might be an instance of teaching music theory as opposed to showing how to use LilyPond, Jan. I've been assuming that our audience is able to read music well enough to want to engrave their own, so the extra material would not really be needed.

jan-warchol commented on uczenie/2 wprowadzenie do LilyPonda.ly:
```diff
@@ -188,9 +174,9 @@
 
-\new Staff \with { \remove "Time_signature_engraver" } {
+  \new Staff \with { \remove "Time_signature_engraver" } {
   \set Staff.printKeyCancellation = ##f
```
There shouldn't be whitespace changes here.  I think it's a bug in Frescobaldi's _format_ tool, and i think it was fixes some time ago - please chech with a latest version.

CPKCampbell replied:
I've formed the habit of running the Format command frequently, and I'll be sure to use the Strip Trailing Whitespace as well.

jan-warchol commented on uczenie/2 wprowadzenie do LilyPonda.ly:
```diff
@@ -243,16 +251,30 @@
 \markup \justify {
-  To create ties (which connect notes so that they constitute onee sound),
-  use tilde character \huge ~ after each note that should be tied.
-   To type a tilde under Windows, you must press simultaneously
```
I agree with the general trend to make the introduction os-independent, but my impression is that producing tilde on Windows is very confusing and non-savvy users (who are an important part of the target audience) will fail without a hint.

CPKCampbell replied:
Isn't it more a function of the keyboard layout, Jan? Again, I'm thinking our audience is more likely to be comfortable with computer applications than not, and definitely more familiar with their keyboard layout. Most keyboards would have it as shift - (whatever that key is left of the 1), but that key on my system has 5 characters, so I'd rather not get too specific.

jan-warchol commented on uczenie/2 wprowadzenie do LilyPonda.ly:
```diff
@@ -433,7 +442,8 @@ Shift and press the tilde (next to number 1) and then a space.
 \markup \typewriter "\\tempo \"Con moto\" 4=110"
 \relative f' {
-  \tempo "Con moto" 4=110
+  \tempo "Con moto" 4=1
```
this looks like a typo :)

CPKCampbell replied:
Well, we don't want to rush into things, do we?

jan-warchol commented on uczenie/2 wprowadzenie do LilyPonda.ly:
```diff
@@ -462,17 +472,25 @@ Shift and press the tilde (next to number 1) and then a space.
+%{
+convert-ly (GNU LilyPond) 2.19.8  convert-ly: Processing `'...
+Applying conversion: 2.17.97, 2.18.0, 2.19.2, 2.19.7
+%}
```
please remove convert-ly information.

CPKCampbell replied:
Done.

jan-warchol commented:
Hi,
it's great to see you working on this!  I added a few comments.

CPKCampbell committed:
First round of comments from Janek

Re-wrote section on editors
Changed explanation and example of relative pitch
Changed "apostrophe" to "single quote" for consistency
Fixed minor typos and extra spaces
Ran Frescobaldi Format and Strip Trailing Whitespace

CPKCampbell commented:
On 14-06-24 11:36 AM, Janek Warchoł wrote:

> Hi,
> it's great to see you working on this! I added a few comments.

One endeavours to give satisfaction! I've updated the file, but I'm not 
sure about whether I need to issue a new pull request, and if so, how, 
or whether you get notified automagically.

Cheers,
Colin

## 

I've learned that you shouldn't go through life with a catcher's mitt on both hands.
You need to be able to throw something back.
-Maya Angelou, poet (1928- )
